modules/standard/Math.chpl

```diff
@@ -47,107 +41,138 @@ handling in the Math module.  The default behavior is as if the macro
+   prefix.
+
+.. include:: AutoMath.rst
+  :start-line: 7
+  :start-after: Automatically included Math symbols
+  :end-before: .. warning::
+
+Non-Automatically Available Symbols
+-----------------------------------
+
 .. _math-constants:
 
 Constants
----------
-:param:`e`
-:param:`log2E`
-:param:`log10E`
-:param:`ln2`
-:param:`ln10`
-:param:`pi`
-:param:`halfPi`
-:param:`quarterPi`
-:param:`reciprPi`
-:param:`twiceReciprPi`
-:param:`twiceReciprSqrtPi`
-:param:`sqrt2`
-:param:`reciprSqrt2`
+^^^^^^^^^
+:param:`~Math.e`
+:param:`~Math.log2E`
+:param:`~Math.log10E`
+:param:`~Math.ln2`
+:param:`~Math.ln10`
+:param:`~Math.pi`
+:param:`~Math.halfPi`
+:param:`~Math.quarterPi`
+:param:`~Math.reciprPi`
+:param:`~Math.twiceReciprPi`
+:param:`~Math.twiceReciprSqrtPi`
+:param:`~Math.sqrt2`
+:param:`~Math.reciprSqrt2`
 
 .. _math-trigonometry:
 
 Trigonometric Functions
------------------------
-:proc:`acos`
-:proc:`acosh`
-:proc:`asin`
-:proc:`asinh`
-:proc:`atan`
-:proc:`atan2`
-:proc:`atanh`
-:proc:`cos`
-:proc:`cosh`
-:proc:`sin`
-:proc:`sinh`
-:proc:`tan`
-:proc:`tanh`
+^^^^^^^^^^^^^^^^^^^^^^^
+:proc:`~Math.acos`
+:proc:`~Math.acosh`
+:proc:`~Math.asin`
+:proc:`~Math.asinh`
+:proc:`~Math.atan`
+:proc:`~Math.atan2`
+:proc:`~Math.atanh`
+:proc:`~Math.cos`
+:proc:`~Math.cosh`
+:proc:`~Math.sin`
+:proc:`~Math.sinh`
+:proc:`~Math.tan`
+:proc:`~Math.tanh`
 
 .. _math-log:
 
 Log Functions
--------------
-:proc:`ln`
-:proc:`log`
-:proc:`log10`
-:proc:`log1p`
-:proc:`logBasePow2`
-:proc:`log2`
+^^^^^^^^^^^^^
+:proc:`~Math.ln`
+:proc:`~Math.log`
+:proc:`~Math.log10`
+:proc:`~Math.log1p`
+:proc:`~Math.logBasePow2`
+:proc:`~Math.log2`
 
 .. _math-exponential:
 
 Exponential Functions
----------------------
-:proc:`exp`
-:proc:`exp2`
-:proc:`expm1`
-:proc:`ldExp`
+^^^^^^^^^^^^^^^^^^^^^
+:proc:`~Math.exp`
+:proc:`~Math.exp2`
+:proc:`~Math.expm1`
+:proc:`~Math.ldExp`
 
 .. _math-rounding:
 
 Rounding
---------
-:proc:`divCeil`
-:proc:`divCeilPos`
-:proc:`divFloor`
-:proc:`divFloorPos`
-:proc:`nearbyint`
-:proc:`rint`
+^^^^^^^^
+:proc:`~Math.divCeil`
+:proc:`~Math.divCeilPos`
+:proc:`~Math.divFloor`
+:proc:`~Math.divFloorPos`
+:proc:`~Math.nearbyint`
+:proc:`~Math.rint`
 
 .. _math-gamma:
 
 Gamma Functions
----------------
-:proc:`gamma`
-:proc:`lnGamma`
+^^^^^^^^^^^^^^^
+:proc:`~Math.gamma`
+:proc:`~Math.lnGamma`
 
 .. _math-error:
 
 Error Functions
----------------
-:proc:`erf`
-:proc:`erfc`
+^^^^^^^^^^^^^^^
+:proc:`~Math.erf`
+:proc:`~Math.erfc`
 
 .. _math-algorithms:
 
 Algorithms
-----------
-:proc:`gcd`
+^^^^^^^^^^
+:proc:`~Math.gcd`
 
 .. _math-bessel:
 
 Bessel Functions
-----------------
-:proc:`j0`
-:proc:`j1`
-:proc:`jn`
-:proc:`y0`
-:proc:`y1`
-:proc:`yn`
+^^^^^^^^^^^^^^^^
+:proc:`~Math.j0`
+:proc:`~Math.j1`
+:proc:`~Math.jn`
+:proc:`~Math.y0`
+:proc:`~Math.y1`
+:proc:`~Math.yn`
+
+.. _automath-constant-and-function-definitions:
+
+Constant and Function Definitions for AutoMath
```

Comment:
Let's say "Automatically Included Constant and Function Definitions" instead of mentioning `AutoMath`, since all other references to it should go away